tools/ci-cdk/canary-lambda/src/latest/transcribe_canary.rs

```diff
@@ -64,18 +66,19 @@ pub async fn transcribe_canary(
         }
     }
 
-    if expected_transcribe_result != full_message.trim() {
-        Err(CanaryError(format!(
+    let dist = edit_distance(&expected_transcribe_result, full_message.trim());
+    let max_edit_distance = 10;
+    if dist > 5 {
```

Comment:
Did you mean to use `max_edit_distance` in the condition?

Reply:
yes, fixed